### ISSUE TITLE
As a super uber admin why can I ..

### ISSUE THREAD

**Issue by msg4real** [magfest/ubersystem#2964]
I should not be able to make any attendees and/or minors uber admin accounts.. it should veryavery that you want to do something that is most likely incorrect

**Comment by ShevaDas** [comment]
Registration at least definitely needs to make admin accounts for attendees (at least those who have a volunteer ribbon, and adding in a check for the volunteer ribbon to the admin account creation seems like a lot of overhead). I also vaguely recall some special circumstance where minors are allowed to volunteer with a guardian.

**Comment by msg4real** [comment]
ShevaDas i was referring  to ma de admin accounts to manage uber ones that dh and some mangers have. NOT to create staff checklist access permission.


I was suggesting a warning vs a ban on it as it's uncommon for attendee volunteers to have one.

**Comment by ShevaDas** [comment]
I'm not sure I'm parsing you correctly. Registration volunteers have Uber admin accounts just like DHs have Uber admin accounts. The only thing that changes is the permissions.

**Comment by msg4real** [comment]
yes, but I am talking about before the logic changes. ONsite we allow things that aren't allowed offsite 

example making minors volunteers..  and yes reg people do have admin accounts you are right
